packages/viewer/src/components/viewer-pin-label/viewer-pin-label.tsx

```diff
@@ -187,6 +193,8 @@ export class VertexPinLabel {
             minWidth: this.computeMinWidth(),
             maxWidth: this.computeMaxWidth(),
             maxHeight: this.computeMaxHeight(),
+            'border-color': primaryColor,
```

Comment:
Minor nit: could this just be `borderColor: primaryColor` to follow the same camelCase pattern of the other styles?